integration/spark/src/main/scala/org/apache/spark/sql/parser/CarbonSparkSqlParserUtil.scala

```diff
@@ -215,10 +215,6 @@ object CarbonSparkSqlParserUtil {
       table.identifier.table.toLowerCase()
     )
     val tableInfo = if (isExternal) {
-      if (partitionColumnNames.nonEmpty) {
```

Comment:
Why can we remove it?
do we support this now?

Reply:
This scenario, already works fine in cluster. I have tested in Spark-Hadoop cluster. Local mode only uses this flow. So, better to keep same for all environment